src/components/Thumbnail/Thumbnail.js

```diff
@@ -15,17 +15,23 @@ class Thumbnail extends React.PureComponent {
     currentPage: PropTypes.number,
     pageLabels: PropTypes.array.isRequired,
     canLoad: PropTypes.bool.isRequired,
+    isSelected: PropTypes.bool,
     onLoad: PropTypes.func.isRequired,
     onCancel: PropTypes.func.isRequired,
     onRemove: PropTypes.func.isRequired,
     updateAnnotations: PropTypes.func,
     closeElement: PropTypes.func.isRequired,
+    onDragStartCallback: PropTypes.func,
+    onDragOverCallback: PropTypes.func,
```

Comment:
I prefer `onDragStart` and `onDragOver`. I think it's common that variables start with `on` are callbacks, so we can probably skip the `callback` part? :)

Reply:
I have renamed these